osd/customer_pdb_preventing_drain.json

```diff
@@ -0,0 +1,8 @@
+{
+ "severity": "Warning",
+ "service_name": "SREManualAction",
+ "cluster_uuid": "${CLUSTER_UUID}",
+ "summary": "Action required: Pod Disruption Budget preventing Node Drain",
+ "description": "Your OpenShift Dedicated cluster is attempting to drain a node but there is a pod disruption budget set that is preventing the drain. The OSD SRE team has identified the pod as ${POD} running in ${NAMESPACE}. Please manually re-schedule this pod so that the node can drain. For more information on pod disruption budgets please see https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-configuring.html#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring",
```

Comment:
Suggesting a little word pruning: 
"Your OpenShift Dedicated cluster is attempting to drain a node but there is a pod disruption budget preventing the drain. The OSD SRE team has identified the pod as ${POD} running in ${NAMESPACE}. Please re-schedule this pod so that the node can drain. For more information on pod disruption budgets please see https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-configuring.html#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring"